Capfile

```diff
@@ -26,5 +26,5 @@ require 'capistrano/passenger'
 require "whenever/capistrano"
 require "airbrussh/capistrano"
 
-# Load custom tasks from `lib/capistrano/tasks` if you have any defined
-Dir.glob('lib/capistrano/tasks/*.rake').each { |r| import r }
+# Loads custom tasks from `lib/capistrano/tasks' if you have any defined.
+Dir.glob('lib/capistrano/tasks/*.cap').each { |r| import r }
```

Comment:
It literally makes no difference aside from making a clear distinction between what are capistrano tasks and what are rake tasks. Dependent on what's in this file, Capistrano will take either `.rake` or `.cap`
Personally I like separating Rake tasks and Cap tasks like this